crates/types/src/lib.rs

```diff
@@ -868,8 +883,12 @@ impl TypeTrace for WasmCompositeType {
 /// A concrete, user-defined (or host-defined) Wasm type.
 #[derive(Debug, Clone, Eq, PartialEq, Hash, Serialize, Deserialize)]
 pub struct WasmSubType {
-    // TODO: is_final, supertype
-    //
+    /// Whether or not this type can be the supertype of any other type.
+    pub is_final: bool,
```

Comment:
Interpreted literally I think these docs are backwards since `true` means it can't be a supertype of anything else. Mind rewording the other way around?